### ISSUE TITLE
Space Center scene camera on opposite side of world

### ISSUE THREAD

**Issue by gordonfpanam** [Kopernicus/Kopernicus#188]
I'm running into an old problem GregroxMun ran into with his Alien Space Programs worlds. The Space Center scene's camera is not focused on the Space Center. My fork is here:

https://github.com/gordonfpanam/Alien-Space-Programs

The Space Center scene's camera isn't at the Space Center in the Duna-based config, and after placing a craft 180 degrees away, that craft appears in the Space Center scene ready to be recovered. It might take a scene switch to have it appear though.

Oddly, the Eve-based config doesn't exhibit this problem. I've poured over the configurations to look for discrepancies between the Eve and Duna versions, and I can't find anything that triggers this behaviour. On a lark I even tried PostSpawnOrbit{} to 're-parent' the home world around the Sun after seeing issue #185 and thinking that might help.

Yes, the readme tells me that re-parenting Kerbin can break the Space Center scene, but in both the Duna and Eve cases it just changes orbit around the original parent.

This behaviour occurs with the KSP 1.2.2 and KSP 1.3 editions of Kopernicus. The same configurations seem to work and behave the same in both editions.

**Comment by Sigma88** [comment]
I think this is another manifestation of the bug that causes #186
Try deleting "Bin" and ser if that fixes your issue

If that works then you will need to change the template for bin from "kerbin" to "laythe"

**Comment by gordonfpanam** [comment]
So just don't create Bin, then, at least for testing. I'm at work but I can try this when I get home today.

Bin is Kerbin renamed in these configurations, followed by creating a substitute Kerbin with the properties of whatever world I'm trying to start with, so Bin would have a lot of unwanted properties such as comm stations and a KSC.

I take it these Module Manager patches of Kopernicus{} occur before the system gets built then? Never thought of that.

**Comment by Sigma88** [comment]
yes, specifically make sure you have no planets with a "Kerbin" template other than the homeplanet

also, you can now use "displayName = Duna" instead of "cbNameLater = Duna"

but you need to put it inside the Properties node instead of just in the Body node

**Comment by gordonfpanam** [comment]
And... that did it! First time I've managed to see the Duna Space Center actually on Duna's surface.

I'll do a new pre-release in a few moments. Right now Laythe is substituting for Kerbin as you suggested, but I have to figure out how to clone Kerbin without using the template. The first couple of times I tried copying Kerbin's surface properties to another world, I had craft 'flying' while landed with a surface speed of zero, and the terrain was all jagged and wrong. But that's another matter.
